### PR TITLE
backup: describe how to restore

### PR DESCRIPTION
**Pull request by koying**
## Proposed change
Clarify how to restore a backup made in Container or Core

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/backup.markdown [hunk 1]
```diff
@@ -69,3 +69,18 @@ automation:
       alias: "Create backup now"
       service: backup.create
 ```
+
+## Restoring a backup
+
+<div class="note">
+
+If you use Home Assistant Operating System or Home Assistant Supervised, [restore functionality is already built-in](/common-tasks/os/#restoring-a-backup).
```
Comment (c0ffeeca7) [inline source/_integrations/backup.markdown]:
```suggestion
If you use Home Assistant Operating System or Home Assistant Supervised, [the restore functionality is already built-in](/common-tasks/os/#restoring-a-backup).
```

File: source/_integrations/backup.markdown [hunk 2]
```diff
@@ -69,3 +69,18 @@ automation:
...
+
+</div>
+
+Backups created via the integrations are located in you `/config/backups` directory. Home Assistant Container installation will typically mount this directory via `docker-compose.yml` or `docker run` to a directory of they choice.
```
Comment (c0ffeeca7) [inline source/_integrations/backup.markdown]:
```suggestion
Backups created via the **Backup** integration are located in your `/config/backups` directory. The Home Assistant Container installation will typically mount this directory via `docker-compose.yml` or `docker run` to a directory of your choice.  
```

File: source/_integrations/backup.markdown [hunk 3]
```diff
@@ -69,3 +69,18 @@ automation:
...
+## Restoring a backup
+
+<div class="note">
+
+If you use Home Assistant Operating System or Home Assistant Supervised, [restore functionality is already built-in](/common-tasks/os/#restoring-a-backup).
+
+</div>
+
+Backups created via the integrations are located in you `/config/backups` directory. Home Assistant Container installation will typically mount this directory via `docker-compose.yml` or `docker run` to a directory of they choice.  
+There is currently no builtin way to restore a backup. However, a Home Assistant backup is just a tar file of the `/config` directory, plus some metadata. To manually restore a backup, one can do:
```
Comment (c0ffeeca7) [inline source/_integrations/backup.markdown]:
```suggestion
For Container and Core installations, there is currently no built-in way to restore a backup. However, a Home Assistant backup is just a tar file of the `/config` directory, plus some metadata. To manually restore a backup, you can use the following:
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @koying :+1:

File: source/_integrations/backup.markdown [hunk 4]
```diff
@@ -69,3 +69,18 @@ automation:
...
+
+If you use Home Assistant Operating System or Home Assistant Supervised, [the restore functionality is already built-in](/common-tasks/os/#restoring-a-backup).
+
+</div>
+
+Backups created via the **Backup** integration are located in your `/config/backups` directory. The Home Assistant Container installation will typically mount this directory via `docker-compose.yml` or `docker run` to a directory of your choice.  
+For Container and Core installations, there is currently no built-in way to restore a backup. However, a Home Assistant backup is just a tar file of the `/config` directory, plus some metadata. To manually restore a backup, you can use the following:
+
+```shell
+$ tar -xOf <backup_tar_file> "./homeassistant.tar.gz" | tar --strip-components=1 -zxf - -C <restore_directory>
```
Comment (frenck) [inline source/_integrations/backup.markdown:85]: Not sure why our CI passes here, but the leading `$` should not have been added and must be removed.

Can you open up a PR to correct this?

../Frenck

Comment (c0ffeeca7) [reply source/_integrations/backup.markdown:85]: sure, will do! Thank you for the catch!
#31748

**Pull request merged**